button.css

```diff
@@ -0,0 +1,58 @@
+.btn-container{
+    width: 525px;
+    height: 91px;
+    display: flex;
+    gap: 40px;
+    flex-wrap: wrap;
+}
+
+.btn-container .primay-btn{
+    padding: 9px 10px;
```

Comment:
Try to make your button sizes consistent, Just change the functionality and appearance on different use cases like "outline-button". In primary-btn case you are changing the width of button based on content and padding and in other cases you have pre-defined fixed width and height.

Reply:
ok sure